pages/accounts/signin/assessee.tsx

```diff
@@ -0,0 +1,8 @@
+import { Login } from "@components/features/Login/Login";
```

Comment:
Hey Alex, no comment on the code, but I would like to suggest that the directory could be named login rather than signin since it will also be consistent with naming of the components.

Reply:
Oops, I forgot...
Thanks Rashad for pointing it out, I forgot to use the barrel for our imports. I have made the barrels though.
I'll refactor it for the next commit.

Reply:
Okay Alex, actually I was referring to the file based routing of the pages, instead of pages/accounts/signin I think it would be better if it was pages/accounts/login for better clarity and consistency